src/soap/serializers/query.js

```diff
@@ -9,23 +9,30 @@ export function queryInfoEnd() {
 }
 
 export function queryInfoType({ date = new Date(), method, parameters }) {
-  if (method === 'GetSitesObject') {
-    method = 'GetSites'
-  } else if (method === 'GetVariablesObject') {
-    method = 'GetVariableInfo'
+  switch (method) {
+    case 'GetSitesObject':
+      method = 'GetSites'
+      break
+    case 'GetVariablesObject':
+      method = 'GetVariableInfo'
+      break
   }
+
   return (
     `<creationTime>${date.toISOString()}</creationTime>` +
     `<criteria MethodCalled="${method}">` +
     parameters
-      .map(parameter =>
-        !parameter[1]
-          ? ''
-          : `<parameter name="${parameter[0]}" value="${encodeXML(
-              parameter[1] + ''
-            )}"/>`
+      .map(
+        parameter =>
+          `<parameter name="${parameter[0]}" ${
+            parameter[1] ? `value="${encodeXML(parameter[1])}"` : ''
+          }/>`
       )
       .join('') +
     '</criteria>'
   )
 }
+
+export function queryInfoNote({ note, vis = true }) {
```

Comment:
Spell out please,`visible`. Question, why not just use `typeof note === 'string'` then pass `''` and not need the additional visibility option?

Reply:
The reason we need `visible` option, in GetSites/Object without parameter there is a note that is  "ALL Sites(empty request)"